DEVELOPING.md

```diff
@@ -51,6 +52,16 @@ For each example there are 3 modes:
 
 - `/examples/abc.html` points to prod. This file would not reflect your local changes.
 
+## Manual visual testing with gemini
```

Comment:
this may be my mistake. The root url should be localhost. It does not need to be ngrok. That was a change I was testing at the end, and it should not have been checked in. I pushed up the correct url that refers to localhost and it should be working now.